README.md

```diff
@@ -162,6 +162,7 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [How To Generate and Store Map Previews in Ruby with Appwrite](https://dev.to/oliverpham/how-to-generate-and-store-map-previews-in-ruby-with-appwrite-43ha)
 * [What is Appwrite and who i setup in my project?](https://dev.to/jordi5107/what-is-appwrite-and-who-i-setup-in-my-project-3af)
 * [How Appwrite real-time can make your CMS changes instant](https://sakshiuppoor.hashnode.dev/how-appwrite-real-time-can-make-your-cms-changes-instant)
+* [Appwrite + JQuery](https://dev.to/leonyangela/build-your-app-with-appwrite-jquery-55ij)
```

Comment:
Great PR! 🤯 A few requests:

1. Would you please update the README in your repo to include instructions about the collection attribute needed?
2. Would you please also update the README to include some screenshots?

Reply:
Hi Steven, sure. I just added the instructions and the screenshots. Thank you :).